addons/calendar/models/calendar_event.py

```diff
@@ -485,6 +496,13 @@ def write(self, values):
         if 'partner_ids' in values:
             values['attendee_ids'] = self._attendees_values(values['partner_ids'])
             update_alarms = True
+            if self.videocall_channel_id:
+                new_partner_ids = []
+                for value in values['partner_ids']:
+                    op = value[0]
+                    if op in (Command.LINK, Command.SET):
+                        new_partner_ids.append(value[1]) if op == Command.LINK else new_partner_ids.extend(value[2])
```

Comment:
```suggestion
                for command in values['partner_ids']:
                    if command[0] == Command.LINK:
                        new_partner_ids.append(command[1])
                    elif command[0] == Command.SET:
                        new_partner_ids.extend(command[2])
```
The semantics for SET also look odd, normally it removes all existing records and adds the new ones. But here partners removed from the event will remain members of the video call (even more so as CLEAR and DELETE are not processed)?

Reply:
My reasoning to implement it like this  is because in recurring events, you might want to delete some partner from an event, but not from all of the events in the recurrence and this would cause him to be removed completely. In case you want to remove someone from the discuss channel, you can go to the discuss app and remove it directly from there.

I'll check with the PO what he thinks about this. I agree it's confusing to use SET and not replace everything.

Reply:
Also, it is possible that I invite someone directly to the discuss channel without having the person invited to the meeting. Using the default SET semantic, we would be deleting these people from the discuss channel as well.